.github/workflows/gnitpick.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Nitpick git commits'
+
+on: [push, pull_request]
+
+env:
+  PYTHON_VERSION: 3.6
+
+jobs:
+  nitpick:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup repository
+        uses: actions/checkout@v2
+
+      - name: Set up python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Install dependencies
+        run: pip install flake8
+
+      - name: Run gnitpick
+        run: python gnitpick.py --verbose
+
+      - name: Lint code
```

Comment:
As this is action for *running Gnitpick*, running other stuff, like linting Python code is IMHO out-of-scope for this action? This makes it much harder to use this action, as different projects might have different needs to eg. globally ignore certain errors etc. :)

Reply:
Yeah, that makes sense. Probably add another action for linting the code?

Reply:
Yes, that would be much better :)